policyengine_us/data/parameters/gov/aca/ca/gra.yaml

```
@@ -0,0 +1,165 @@
+description: ACA geographic rating areas for California.
+metadata:
+  reference:
```

```suggestion
  label: California ACA geographic rating areas
  reference:
  
```
Please add labels to all parameters

---

I was just following the example set by Nikhil in his original ACA implementation, in which parameters have a `name:`.   If that is wrong, why did he do it that way?   I don't see anywhere in the GitHub repo where the "rules" you are describing are specified.   Have I missed a link to that information?

---

Revised description and added label to both the `data/parameters/aca/ca/gra.yaml` and `data/parameters/aca/ca/slspc.yaml` parameter files.

I think all the other parameter files have a label.

---

DONE